README.adoc

```diff
@@ -0,0 +1,15 @@
+# spring-cloud-gcp
+Integration for Google Cloud Platform APIs with Spring
+
+=== Code formatting guidelines
+
+* The directory ./etc/eclipse has two files for use with code formatting, `eclipse-code-formatter.xml` for the majority of the code formatting rules and `eclipse.importorder` to order the import statements.
+
+* In eclipse you import these files by navigating `Windows -> Preferences` and then the menu items `Preferences > Java > Code Style > Formatter` and `Preferences > Java > Code Style > Organize Imports` respectfully.
+
+* In `IntelliJ`, install the plugin `Eclipse Code Formatter`.  You can find it by searching the "Browse Repositories" under the plugin option within `IntelliJ` (Once installed you will need to reboot Intellij for it to take effect).
```

Comment:
No reason to wrap IntelliJ to the code. And brand name is **IntelliJ IDEA**

And this is official open for everyone document, so some typos like `Intellij` aren't appropriate